pkg/lib/queueinformer/queueinformer_operator_test.go

```diff
@@ -78,7 +78,7 @@ func TestOperatorRunChannelClosure(t *testing.T) {
 
 			o.Run(ctx)
 
-			timeout := time.After(time.Second)
+			timeout := time.After(2 * time.Minute)
```

Comment:
is this meant to represent `2*defaultServerVersionInterval` ?

Reply:
Yeah, it probably should be.

Reply:
Fixed